.github/wiki/Deployment.md

```diff
@@ -1,3 +1,32 @@
-# Welcome to Deployment page!
+# Deployment Process
 
-This is an example page.
\ No newline at end of file
+## For iOS
+
+### Guide
+
+- Before deploying the iOS application, we have to prepare the necessary credentials and information:
+
+    - An Apple developer account.
+    - Your application’s bundle id. This could be multiple bundle ids according to the number of application’s flavors.
+    - An application that links to the bundle id.
```

Comment:
What does this mean? Do you mean it's our project build for app store publishing? Is it a redundant thing to note here? 💭

Reply:
Or do you mean we need to setup a new app on App Store Connect? This sentence is quite vague 💭

Reply:
> Or do you mean we need to setup a new app on App Store Connect? This sentence is quite vague 💭

@luongvo yeah, I mean that it's a new app on App Store Connect 😄 Maybe I will rewrite the sentence to `A new app's on App Store Connect that links to the bundle id.` What do you think?

Reply:
@manh-t Sounds better